### ISSUE TITLE
adapting the padding through css

### ISSUE THREAD

**Issue by sebastiendementen** [Rebilly/ReDoc#623]
I can't find a way to change the padding in the ReDoc (currently 40px top & bottom).

I have a css including
```
redoc > div > div.api-content {
   padding: 0px 0px 0px 0px;
   color: red;
}
```
The colod:red is just there to be sure ReDoc is using the css (and indeed I see red fonts :-) ).
However, regarding the padding, the padding top & bottom is still at 40px. If I set left and right padding in my CSS, ReDoc uses it corretcly.

Via the Chrome dev tools, I see something like
```
.bOgKnn {
    padding: 40px 0px;
    padding-top: 40px;
    padding-right: 0px;
    padding-bottom: 40px;
    padding-left: 0px;
}
```
that defines the padding.

==> only the padding-bottom and padding-top seems unchangeable.

**Comment by RomanGotsiy** [comment]
Padding of which element are you trying to adjust? Could you share a screenshot?

**Comment by sebastiendementen** [comment]
The padding as show in the pic 
![image](https://user-images.githubusercontent.com/41316461/44841810-855cb880-ac44-11e8-99b7-cf074fd4ae58.png)

**Comment by RomanGotsiy** [comment]
I will add a theme variable to control that padding.
will let you know here once done

**Comment by sebastiendementen** [comment]
great! does this mean you will add a class to the html tag so we can theme it via standard CSS ?
And overall question, today I do not see class names for the different elements of ReDoc (title, extraDocuments, ...), is it correct ? I see class name that look autogenerated:
`<label role="menuitem" class="sc-kIPQKe -depth1 cYxcnd" type="section"><span class="sc-eXEjpC ndUuo" title="Introduction">Introduction</span></label>`
This makes it difficult to style it via CSS. However, I see there was a concept of theme but could not find much info in the ReDoc docs

**Comment by RomanGotsiy** [comment]
Fixed in 505463f

Added new theme option `sectionVertical` (`40` by default). To change:

```js
ReDoc.init('<spec url>', {
  theme: {
    spacing: {
      sectionVertical: 20
    }
  }
}
```

Will be available in the `2.0.0-alpha.40` coming in 2-3 hours

**Comment by anupsingh-anup** [comment]
still i don't see a option to change only padding-top for h1,h2  and I'm trying to add header at the top and when i click on left menu bar  and it behaves differently. @RomanHotsiy .
